README.md

```diff
@@ -1,25 +1,42 @@
-# Operator for storage monitoring mixin
+# Operator for Storage Monitoring Mixins
 
-This operator takes care of deployment of mixins of storage system in
-kubernetes. The mixins is an abstraction framework that makes it easier
+Monstorak operator takes care of deployment of monitoring mixins for storage systems in
+K8s/Openshift.
```

Comment:
mentioned k8s but isn't going to work.

k8s doesn't have the command: https://github.com/monstorak/monstorak/pull/35/files#diff-04c6e90faac2675aa89e2176d2eec7d8R16

Reply:
@cloudbehl ,
 this can be treated as a separate bug and should be fixed. 'README' still can be same...

Reply:
fixed.